docs/sections/Docs-Type/index.jsx

```diff
@@ -24,6 +24,11 @@ export default class Type extends React.Component {
         </section>
 
         <section className="doc-bottom-space-large">
+          <h1> Link </h1>
+          <a href="/#/type"> Redirect me to Something </a>
```

Comment:
Set link to `javascript:void(0);` because clicking the link would affect the current link (and IMO, would unintentionally put the user to `/home`) since we're using `HashHistory`.